bin/templates/cordova/lib/env/java.js

```diff
@@ -96,10 +96,9 @@ const java = {
         const javacPath = utils.forgivingWhichSync('javac');
         const hasJavaHome = !!environment.JAVA_HOME;
         if (hasJavaHome) {
-            // Windows java installer doesn't add javac to PATH, nor set JAVA_HOME (ugh).
-            if (!javacPath) {
```

Comment:
if javacPath is not used here.

The scope of the const can be smaller. Maybe place the `` const javacPath = utils.forgivingWhichSync('javac');`` after the else { (Line 103)?